application/config/module.config.php

```diff
@@ -247,4 +247,44 @@
             'image/gif'  => 'image',
         ),
     ),
+    'oembed' => array(
+        'whitelist' => array(
```

Comment:
All the regexes here probably need start-of-string (`^`) and end-of-string (`$`) anchors to make sure we're matching the whole URL